lib/client/ruby/tunnel-vision.rb

```diff
@@ -73,20 +93,20 @@ def remote_port
       # Try reading from the options, if specified
       return options[:remote_port] if options[:remote_port]
 
-      # Try to automatically determine this user's port from the ansible play
-      # config file mapping
+      # Try to automatically determine this user's port mapping
+      # A JSON mapping file should be available at
+      #   "https://#{TUNNEL_HOST}/users.json"
 
-      uri = URI(CONFIG_URL)
+      uri = URI("https://#{TUNNEL_HOST}/users.json")
```

Comment:
This client tries to automatically determine the `PORT` number for your username. This is so that you (the user) does not have to remember the port. 

The old approach was to fetch the `raw` version of the config file in this repository. Because of that, I needed this repository to be `public`.

The new (and I think better!) way is to generate the mapping as a JSON file and host it on our server

See: https://pipe.cr-tunnel.xyz/users.json

The client can read this file and determine the port for each user. And then I can make this repository `private`

Reply:
that's great for now, we can work on any improvements later 💯